worlds/hades/__init__.py

```diff
@@ -0,0 +1,255 @@
+            item_name = trap_pool[index]
+            item = HadesItem(item_name, self.player)
+            pool.append(item)
+            index = (index+1) % len(trap_pool)
+            
+        self.multiworld.itempool += pool
+
+    def should_ignore_weapon(self, name):
+        if (self.options.initial_weapon == 0 and name == "SwordWeaponUnlockItem"):
+            return True
+        if (self.options.initial_weapon == 1 and name == "BowWeaponUnlockItem"):
+            return True
+        if (self.options.initial_weapon == 2 and name == "SpearWeaponUnlockItem"):
+            return True
+        if (self.options.initial_weapon == 3 and name == "ShieldWeaponUnlockItem"):
+            return True
+        if (self.options.initial_weapon == 4 and name == "FistWeaponUnlockItem"):
+            return True
+        if (self.options.initial_weapon == 5 and name == "GunWeaponUnlockItem"):
+            return True
+        return False
+
+    def set_rules(self):
+        local_location_table = setup_location_table_with_settings(self.options).copy()
+        set_rules(self.multiworld, self.player, self.calculate_number_of_pact_items(
+        ), local_location_table, self.options)
+
+    def calculate_number_of_pact_items(self):
+        # Go thorugh every option and count what is the chosen level
+        total = int(self.options.hard_labor_pact_amount.value)
+        total += int(self.options.lasting_consequences_pact_amount.value)
+        total += int(self.options.convenience_fee_pact_amount.value)
+        total += int(self.options.jury_summons_pact_amount.value)
+        total += int(self.options.extreme_measures_pact_amount.value)
+        total += int(self.options.calisthenics_program_pact_amount.value)
+        total += int(self.options.benefits_package_pact_amount.value)
+        total += int(self.options.middle_management_pact_amount.value)
+        total += int(self.options.underworld_customs_pact_amount.value)
+        total += int(self.options.forced_overtime_pact_amount.value)
+        total += int(self.options.heightened_security_pact_amount.value)
+        total += int(self.options.routine_inspection_pact_amount.value)
+        total += int(self.options.damage_control_pact_amount.value)
+        total += int(self.options.approval_process_pact_amount.value)
+        total += int(self.options.tight_deadline_pact_amount.value)
+        total += int(self.options.personal_liability_pact_amount.value)
+        return total
+
+    def create_item(self, name: str) -> Item:
+        return HadesItem(name, self.player)
+
+    def create_regions(self):
+        local_location_table = setup_location_table_with_settings(self.options).copy()
+        create_regions(self, local_location_table)
+
+    def fill_slot_data(self) -> dict:
+        slot_data = {
+            'seed': \
+                "".join(self.multiworld.per_slot_randoms[self.player].choice(string.ascii_letters) for i in range(16))
```

Comment:
```suggestion
                "".join(self.random.choice(string.ascii_letters) for i in range(16))
```
Use the world's random object instead of per_slot_randoms (which is deprecated)